src/main/java/connectingstar/tars/user/command/UserService.java

```diff
@@ -0,0 +1,4 @@
+package connectingstar.tars.user.command;
+
+public class UserService {
```

Comment:
```suggestion
public class UserCommandService {
```
서비스 클래스의 이름은 데이터의 상태 변경하는 서비스는 Command, 데이터를 조회하는 서비스는 Query를 사용해서 이름을 작성하시면 됩니다.

Reply:
넵 감사합니다 ! 근데 저렇게 저거 취소하고 저렇게 하세요 ~ 저 코드 어떻게 한건지 디코로 나중에 알려주실 수 있나여 코드리뷰 짱이다